### PR TITLE
Towards #56 - Update end-to-end-via-mas-and-emas.md

### LINKED ISSUE #56: Homogenize end-to-end examples to mimic main2.md
Right now some of the end-to-end examples don't quite follow the main2.md flow. 

Goal of this issue: get them to follow it
- in particular: steps 2 and 3. In particular, measuring the test_nmse at the end of step 3
- step 1 can simply point to main2.md step 1. Step 4 can simply point to main2.md step 4

Each example needs to produce predictions for 1h, 2h, ..., 12h into the future. Not just e.g. 1h into the future, or just 12h.
- Incorrect: code split the data in training and test and then select 12 samples of the test set as 12 hours predictions 

READMEs to cover:
- [x] [Jaime] Simple uses ETH data from Ocean, and linear models. https://github.com/oceanprotocol/predict-eth/commit/09b121786159e7f50a0d07b43410661185d7257a
- [x] [Jaime] Model comparison for direct prediction 1-12 hours
- ~~[JP] Gentle introduction~~
- ~~[JP] Medium level: MA & EMA. 
  - Ensure that predict 1, 2, .., 12h into future. And, don't use samples of the test set as predictions~~

### PR DESCRIPTION
**Pull request by juanpablovs**
Towards #56 .

Changes proposed in this PR:

- MA and EMA example is homogenized with main2
- Adding the file that will be submitted to the judges
- Adding "setup" and "what the judges will do"

### CODE REVIEW

**Comment by trentmc** [comment]
> Fixes #56 .

FYI, if you merge a PR with description saying "Fixes #__", then github will auto-close that issue.

_This_ PR is only one of the sub-tasks in the issue. You don't want github to auto-close the issue.

The practice that I follow for such cases is to say: "Towards #__".

I changed the PR description accordingly.

**Comment by trentmc** [comment]
Here's my review so far.

----

**Key thing: test data needs to be from 24h, 23h, .., 12h in the past. And your model can only learn from data older than that. The README needs to get fixed accordingly.**

_Details:_ 

#56 includes:

- "Each example needs to produce predictions for 1h, 2h, ..., 12h into the future. Not just e.g. 1h into the future, or just 12h."
- "Incorrect: code split the data in training and test and then select 12 samples of the test set as 12 hours predictions"

This PR needs to get updated such that the README meets those points.

Specifically, right now, it's grabbing the most recent 500 points from binance, then randomly selecting training and testing from those. This is incorrect: it means the model is interpolating from future points, rather than extrapolating to the future.
```python
data_ETH = ccxt.binance().fetch_ohlcv('ETH/USDT', '1h')
...
X_train, X_test, y_train, y_test = train_test_split(X_selected, y, shuffle=True, train_size=0.3)
```

Section 3.3 of main2.md is how you should be getting your test data, and calculating nmse. It gets data from 24h ago, until 12h ago. **Use it.**

And then your model can train on any data, **as long as it's >= 24h old.**

---- 

Other points:
- it should be obvious to readers to see how this README expands upon the steps in main2.md. Therefore, this README's section numbers should correspond to section numbers in main2.md. 
- this README has its own implementation of `calc_nmse()`. That's duplicate; remove it.

**Review by trentmc — CHANGES_REQUESTED** [review]
Changes requested: see [previous comment](https://github.com/oceanprotocol/predict-eth/pull/59#issuecomment-1331959588)
